example_configs/activemq.yml

```diff
@@ -0,0 +1,48 @@
+hostPort:
```

Comment:
You can remove the empty bits, they'll cause confusion

Reply:
Thanks removed =)